src/modules/hide_vodcasts/index.js

```diff
@@ -0,0 +1,32 @@
+const $ = require('jquery');
+const settings = require('../../settings');
+const watcher = require('../../watcher');
+
+const VODCAST_SELECTOR = '.live-channel-card .tw-pill';
+const CARD_SELECTOR = '.live-channel-card';
+
+class HideVodcastsModule {
+    constructor() {
+        settings.add({
+            id: 'hideVodcasts',
+            name: 'Hide Vodcasts',
+            defaultValue: false,
+            description: 'Hide Vodcasts from directory listings'
+        });
+        settings.on('changed.hideVodcasts', value => value === true ? this.hide() : this.show());
+
+        watcher.on('directory.vodcast', () => this.hide());
```

Comment:
`.hide` should hide a single vodcast emitted from the watcher, not target every vodcast on the directory